src/app/parcels/ParcelsPage.tsx

```diff
@@ -322,9 +322,14 @@ const ParcelsPage: React.FC<{}> = () => {
         lastStatus: rowToLastStatusColumn,
     };
 
-    const onParcelTableRowClick = (row: Row<ParcelsTableRow>): void => {
-        setSelectedParcelId(row.data.parcelId);
-        router.push(`/parcels?${parcelIdParam}=${row.data.parcelId}`);
+    const onParcelTableRowClick = (row: Row<ParcelsTableRow>, event: React.MouseEvent): void => {
+        const target = event.target as HTMLDivElement;
+        if (target.children.length > 0 && target.children[0].tagName === "INPUT") {
```

Comment:
What are we really checking here?

Reply:
We are checking that the table cell that was clicked has children and the first child is an input. The row has two cells that have children elements (cell 1 and cell 5), the others do not have children.

![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/160640328/5495483c-4729-4286-8aa7-5721c6c5cfbc)
* I really want to check if data-column-id is equal to 1 but not sure how to achieve this in an elegant way. Any suggestions?

Reply:
Maybe checking id begins with "cell-1"?